Loop/Managers/DeviceDataManager.swift

```diff
@@ -328,18 +334,9 @@ final class DeviceDataManager {
         $isClosedLoopAllowed
```

Comment:
`isClosedLoopAllowed` is another app-level property, that is probably more a LoopAppManager responsibility.

Reply:
OK. I'll move it up there.